packages/chakra-ui/src/components/ui/select.tsx

```diff
@@ -0,0 +1,127 @@
+'use client';
+
+import type { CollectionItem } from '@chakra-ui/react';
+import { Select as ChakraSelect, Portal } from '@chakra-ui/react';
+import { CloseButton } from './close-button';
+import * as React from 'react';
```

Comment:
Two things. First, put the react imports up top. Secondly, don't use the `import * as React from 'react';` pattern. Instead import only that which you need. Like:
`import { ReactNode, RefObject, forwardRef } from 'react'`;